package dwalldorf.achallenge.rest.exception;

public class NotFoundException extends RuntimeException {
}
